Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -96,50 +97,13 @@
 {
     ALWAYS_LOG(LOGIDENTIFIER);
 
+    configureParser(m_parser);
     if (isMainThread()) {
         if (RefPtr webmParser = dynamicDowncast<SourceBufferParserWebM>(m_parser)) {
```

Comment:
`m_parser` can change when switching, should we redo that check every time we update it?

Reply:
That's really just to apply a quirk; it's non-standard, allowing sites to do the wrong thing under very particular circumstances. There's never going to use changeType or even MSE in a worker.